### ISSUE TITLE
Add TTS on web

### ISSUE THREAD

**Issue by ztefanie** [digitalfabrik/integreat-app#2918]
**Is your feature request related to a problem? Please describe.**
As a analphabetic user/less fluent reader I want to have the possibility to get texts read out loud such that I can still use the app and understand the content. This feedback was given the target group contact team

**Describe the solution you'd like**
 Add an option in the UI to read the current text: See design [here](https://www.figma.com/design/cA4F2MwHs2LNGviOWUjfE9/Integreat-Tickets-Frontend?node-id=3358-3&t=mEcwdGGYbx4HVbjl-1)
 This should work for categories
 It should work in all supported languages (if possible)

Leading platforms prioritize accessibility and user control over imposing character-based restrictions. I would recommend making the tts functionality available for all third level pages, providing a consistent experience for our users.

We may use these [library](https://github.com/morganney/tts-react) or search for a better option.

- We only read texts of third level categories
- target group for this feature are people with basic reading skills and/or basic language skills, not for people who cannot read at all
- should be available offline
- we do not want to read "lists with 3 elements" or stuff like this. We expect user to have basic reading skills and read the text while the tts reads it.
- we aim to support web and native, but we will start with native
- pausing should be possible
- we want to have this feature enabled by default, but design should not be annoying for people who do not want to this feature or add an easy way to turn it off / hide it.
- this feature should only be available for integreat, not for malte or aschaffenburg

**Additional context**
- See discussion this "meta/story" issue https://github.com/digitalfabrik/integreat-app/issues/2821
- See additional information on the motiviation [here](https://openproject.tuerantuer.org/projects/community-feedback/work_packages/3171/activity)
